app/src/main/java/org/hackillinois/android/App.java

```diff
@@ -13,9 +16,11 @@
 
 public class App extends Application {
     private static API api;
+    private static Database database;
 
     public void onCreate() {
         super.onCreate();
+        database = Room.databaseBuilder(getApplicationContext(), Database.class, "local-db").fallbackToDestructiveMigration().build();
```

Comment:
When using the builder pattern, you should put each additional method call on a new line